### PR TITLE
[13.0][ADD] sale_report_delivered_elaboration: New glue module to display cost elaboration on sale report delivered

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT32439

ping @chienandalu  @CarlosRoca13 

depends on:
- [x] #124 
- [x] https://github.com/OCA/margin-analysis/pull/118

### CODE REVIEW

**Review by chienandalu — APPROVED** [review]
Code review, not tested :+1:

File: sale_report_delivered_elaboration/reports/sale_report.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        select_str = super()._select()
+        select_str += """
+            , sum(signed_qty * unsigned_elaboration_cost) AS elaboration_cost
+        """
+        return select_str
+
+    def _sub_select(self):
+        sub_select_str = super()._sub_select()
+        sub_select_str += """
+            , COALESCE(sm.product_uom_qty, 0.0) * sol.elaboration_cost_price /
```
Comment (CarlosRoca13) [inline sale_report_delivered_elaboration/reports/sale_report.py:22]: When I try to install the module locally, I'm getting the error `Error: column sol.elaboration_cost_price does not exist`.

Are the dependencies correct?

**Review by CarlosRoca13 — CHANGES_REQUESTED** [review]

**Review by CarlosRoca13 — APPROVED** [review]
LGTM

**Comment by CarlosRoca13** [comment]
The dependencies are merged, please rebase

**Comment by sergio-teruel** [comment]
@CarlosRoca13 Rebase done!

**Pull request merged**